tests/arp/test_stress_arp.py

```diff
@@ -85,9 +85,6 @@ def test_ipv4_arp(duthost, garp_enabled, ip_and_intf_info, intfs_for_test,
         loop_times -= 1
         add_arp(ptf_intf_ipv4_hosts, intf1_index, ptfadapter)
 
```

Comment:
Suggest to check if arp entry has beed added

Reply:
done